Formula/web100clt.rb

```diff
@@ -24,6 +18,8 @@ class Web100clt < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "99e17a9830b0b6458263ce8851ed8c8f98d8c2b6bd4b8998e7e82fe88dbaf918"
   end
 
+  deprecate! date: "2022-10-11", because: :deprecated_upstream
```

Comment:
Not entirely sure about `web100clt` formula, but the NDT project is dead based on upstream homepage https://software.internet2.edu/ndt/

> NDT is no longer supported because the Web100 patches to the Linux kernel will not work on modern kernels (3.x and later). A follow-on project called Web10G was never completed. This documentation remains available for historical purposes only.